AppMakefile.mk

```diff
@@ -128,6 +128,7 @@ $$(BUILDDIR)/$(1)/$(1).elf: $$(OBJS_$(1)) $$(LIBS_$(1)) $$(LEGACY_LIBS_$(1)) $$(
 	    -Xlinker --defsym=KERNEL_HEAP_SIZE=$$(KERNEL_HEAP_SIZE)\
 	    -T $$(LAYOUT)\
 	    -nostdlib\
+	    $(LINK_LIBS)\
```

Comment:
This should be `$$(LINK_LIBS_$(1))` so that the make process appends the correct architecture to the end of it